.github/workflows/kbs_build_and_push.yaml

```diff
@@ -0,0 +1,35 @@
+name: Build and Push kbs Image
+
+on:
+  push:
+    branches:
+      - main
+      - kartikjoshi21/kbs-image
+
+jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Get Commit SHA
+      id: commit_sha
+      run: echo "::set-output name=sha::$(git rev-parse --short ${{ github.sha }})"
+
+    - name: Login to GHCR Container Registry
+      uses: docker/login-action@v2
+      with:
+        registry: ghcr.io
+        username: ${{ secrets.repository_owner }}
+        password: ${{ secrets.GHCR_TOKEN }}
+
+    - name: Build Container Image
+      run: |
+        commit_sha=${{ steps.commit_sha.outputs.sha }}
+        DOCKER_BUILDKIT=1 docker build -t ghcr.io/confidential-containers/staged-images:kbs-commit-${commit_sha} --build-arg KBS_FEATURES=coco-as-builtin,openssl,resource,opa . -f docker/Dockerfile --push
```

Comment:
Is `ghcr.io/confidential-containers/staged-images:kbs-commit-${commit_sha}` a convention we have established elsewhere? If not, why not use:

`ghcr.io/confidential-containers/kbs:$sha` or `ghcr.io/confidential-containers/staged-images/kbs:$sha` instead?